### ISSUE TITLE
Feature Request: Allowing captive portal

### ISSUE THREAD

**Issue by zombiestruck** [Snapmaker/Snapmaker2-Controller#33]
The wireless network I use has a captive portal meaning I cannot connect to the machine. This combined with wpa2-enterprise would make it a lot nicer for ease of use.

**Comment by wlcable** [comment]
I agree, at work the only option I have to get internet access is by logging into a webpage after connecting to the open WiFi.
